README.adoc

```diff
@@ -70,21 +70,25 @@ Similarly to `GCP Messaging`, `GCP Storage` contains the Google Cloud Storage su
 If you're a Maven user, add our BOM to your pom.xml `<dependencyManagement>` section.
 This will allow you to not specify versions for any of the Maven dependencies and instead delegate versioning to the BOM.
 
+<!--- {x-version-update-start:spring-cloud-gcp:released} -->
+
 [source,xml]
 ----
 <dependencyManagement>
     <dependencies>
         <dependency>
             <groupId>com.google.cloud</groupId>
             <artifactId>spring-cloud-gcp-dependencies</artifactId>
-            <version>4.0.0</version>
+            <version>3.4.2</version>
```

Comment:
I've deliberately changed this to 3.4.2 (last released version). Release please should update this to 4.0.0 in the release pr.